src/api_server/swagger/firecracker.yaml

```diff
@@ -981,8 +981,8 @@ definitions:
       level:
         type: string
         description: Set the level. The possible values are case-insensitive.
-        enum: [Error, Warning, Info, Debug]
-        default: Warning
+        enum: [Error, Warn, Info, Debug, Trace, Off]
```

Comment:
hmm this is a breaking change, no? Could we keep it as Warning (or provide an alias)?

Reply:
Yeah if it supports both, could we just keep Warning forever and don't deprecate it? Sounds like it's not too hard to maintain.

Reply:
If we did remove `Warning` we could remove the code (see below), it would also make the API more consistent, although it may be low priority. I will remove this change to the `firecracker.yml` here as this can be done or discussed later.
```rust
/// This is required since we originally supported `Warning` and uppercase variants being used as
/// the log level filter. It would be a breaking change to no longer support this. In the next
/// breaking release this should be removed (replaced with `log::LevelFilter` and only supporting
/// its default deserialization).
#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
pub enum LevelFilter {
    /// [`log::LevelFilter:Off`]
    #[serde(alias = "OFF")]
    Off,
    /// [`log::LevelFilter:Trace`]
    #[serde(alias = "TRACE")]
    Trace,
    /// [`log::LevelFilter:Debug`]
    #[serde(alias = "DEBUG")]
    Debug,
    /// [`log::LevelFilter:Info`]
    #[serde(alias = "INFO")]
    Info,
    /// [`log::LevelFilter:Warn`]
    #[serde(alias = "WARN", alias = "WARNING", alias = "Warning")]
    Warn,
    /// [`log::LevelFilter:Error`]
    #[serde(alias = "ERROR")]
    Error,
}
impl From<LevelFilter> for log::LevelFilter {
    fn from(filter: LevelFilter) -> log::LevelFilter {
        match filter {
            LevelFilter::Off => log::LevelFilter::Off,
            LevelFilter::Trace => log::LevelFilter::Trace,
            LevelFilter::Debug => log::LevelFilter::Debug,
            LevelFilter::Info => log::LevelFilter::Info,
            LevelFilter::Warn => log::LevelFilter::Warn,
            LevelFilter::Error => log::LevelFilter::Error,
        }
    }
}

impl FromStr for LevelFilter {
    type Err = LevelFilterFromStrError;
    fn from_str(s: &str) -> Result<Self, Self::Err> {
        match s {
            "Off" | "OFF" => Ok(Self::Off),
            "Trace" | "TRACE" => Ok(Self::Trace),
            "Debug" | "DEBUG" => Ok(Self::Debug),
            "Info" | "INFO" => Ok(Self::Info),
            "Warn" | "WARN" | "Warning" | "WARNING" => Ok(Self::Warn),
            "Error" | "ERROR" => Ok(Self::Error),
            _ => Err(LevelFilterFromStrError(String::from(s))),
        }
    }
}
```